programs/cleos/main.cpp

```diff
@@ -197,6 +199,8 @@ void add_standard_transaction_options(CLI::App* cmd, string default_permission =
 
    cmd->add_option("--max-cpu-usage-ms", tx_max_cpu_usage, localized("set an upper limit on the milliseconds of cpu usage budget, for the execution of the transaction (defaults to 0 which means no limit)"));
    cmd->add_option("--max-net-usage", tx_max_net_usage, localized("set an upper limit on the net usage budget, in bytes, for the transaction (defaults to 0 which means no limit)"));
+  
+   cmd->add_option("-n,--delay", delaysec, localized("set the delay_sec seconds, defaults to 0s"));
```

Comment:
Would prefer: `"--delay-sec"` without any short name.

Reply:
@jjnetcn can you please remove the `-n` and rename to `--delay-sec`.

Reply:
ok,i am not familiar with github/git ，can you send another pull request to remove it？tks

Reply:
Feel free to just push a new one and I'll close this one.

Reply:
Make sure to base it on develop branch.

Reply:
i has push another request, pls close this one